util/cron/test-mpicc.bash

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Test CHPL_COMM=none && CHPL_TARGET_COMPILER=mpi-gnu for MPI module testing
+#
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common.bash
+
+echo >&2 module load mpi
+module load mpi
+
+set -x
+: confirm mpi module is loaded
```

Comment:
":" is a shell command that returns "true" and does nothing else
With set -x, ":" shows up on stderr in the same format as all the other shell commands echo'd  when set -x. 
(we want set -x from here on to see the shell commands - but not before here, because source $CWD/common.bash and any module load commands are ugly.)